contracts/contracts/ERC677/mock/token677_receiver_mock.cairo

```diff
@@ -0,0 +1,94 @@
+%lang starknet
+
+from starkware.cairo.common.cairo_builtins import HashBuiltin
+from starkware.cairo.common.bool import TRUE, FALSE
+from starkware.cairo.common.uint256 import Uint256
+from openzeppelin.introspection.erc165.library import ERC165
+
+const IERC677_RECEIVER_ID = 0x4f3dcd
+
+@storage_var
+func token677ReceiverMock_token_sender_() -> (address : felt):
+end
+
+@storage_var
+func token677ReceiverMock_sent_value_() -> (value : Uint256):
+end
+
+@storage_var
+func token677ReceiverMock_token_data_(index : felt) -> (data : felt):
+end
+
+@storage_var
+func token677ReceiverMock_token_data_len_() -> (data_len : felt):
+end
+
+@storage_var
+func token677ReceiverMock_called_fallback_() -> (bool : felt):
+end
+
+@constructor
+func constructor{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}():
+    ERC165.register_interface(IERC677_RECEIVER_ID)
+    token677ReceiverMock_called_fallback_.write(FALSE)
+    return ()
+end
+
+@external
+func onTokenTransfer{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    sender : felt, value : Uint256, data_len : felt, data : felt*
+):
+    with_attr error_message("ERC677: address can not be null"):
+        assert data_len = 3
```

Comment:
what is 3? Please create constant with explanatory name at the beginning of the block

Reply:
Also error says smth about address but assert happens on `data_len`

Reply:
I removed this, it doesn't make sens anymore in this contract